src/open_inwoner/accounts/tests/test_oidc_views.py

```diff
@@ -207,3 +212,44 @@ def test_error_first_cleared_after_succesful_login(
                 response = self.client.get(error_url)
 
                 self.assertEqual(response.status_code, 403)
+
+
+@patch(
+    "mozilla_django_oidc_db.mixins.OpenIDConnectConfig.get_solo",
+    return_value=OpenIDConnectConfig(enabled=True),
+)
+class OIDCConfigTest(WebTest):
+    def test_admin_only_enabled(self, mock_oidc_config):
+        """Assert that the OIDC login option is only displayed for login via admin"""
+
+        config = SiteConfiguration.get_solo()
+        config.openid_admin_only = True
+        config.save()
```

Comment:
Let's pick one of these two patterns to modify the solo objects for testing. So either @patch both or put both in regular code (like SiteConfiguration is now).

If we do the latter then the test case also needs the ClearCachesMixin, or we'll get unstable results because the solo objects are cached in the cache framework (and Django for some reasons doesn't isolate the caches the way it does the database, hence the custom mixin).